src/coreclr/jit/emitarm64.cpp

```diff
@@ -12311,7 +12311,7 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
 
                 // If there are 2 GC vars in this instrDesc, get the 2nd variable
                 // that should be tracked.
-                adr2     = emitComp->lvaFrameAddress(varNum2, &FPbased2, true);
+                adr2     = emitComp->lvaFrameAddress(varNum2, &FPbased2, FPbased);
```

Comment:
I forgot to tell, but can you add a comment about `suppressFPtoSPRewrite` in the method docs of `lvaFrameAddress`? OK to do in separate PR.